### PR TITLE
Exit with 0 when there are no warnings

### LINKED ISSUE #100: Fix a bug with exit with code 1
`dotenv-linter` always exits with code 1 even if there are no warnings.
Need to fix that bug and write some integration tests to cover that case.

Link: https://github.com/mgrachev/dotenv-linter/blob/master/src/main.rs#L18

### PR DESCRIPTION
**Pull request by simPod**
Resolves #100

### CODE REVIEW

**Comment by mgrachev** [comment]
@simPod 👋Thank you for your contribution! 🚀

Could you also add this PR to [CHANGELOG.md](https://github.com/mgrachev/dotenv-linter/blob/master/CHANGELOG.md) (at the top of the list)?

**Comment by simPod** [comment]
@mgrachev 👋 done

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### 🔧 Changed
 - New CLI API: Ability to check multiple directories [#99](https://github.com/mgrachev/dotenv-linter/pull/99)
+- Add exit with the code 0 when there are no warnings [#105](https://github.com/mgrachev/dotenv-linter/pull/105)
```
Comment (mgrachev) [inline CHANGELOG.md]: Please add information about you at the end of this line. For example:
```
- Refactor `Github Actions` and reduce `Docker Image size` [#90](https://github.com/mgrachev/dotenv-linter/pull/90) ([@Macbet](https://github.com/Macbet))
```

**Comment by mgrachev** [comment]
@simPod Excellent work! Thank you! 👍

**Pull request merged**